components/infobox/wikis/brawlhalla/infobox_league_custom.lua

```diff
@@ -0,0 +1,144 @@
+	local args = _league.args
+	if id == 'customcontent' then
+		if not String.isEmpty(args.player_number) or not String.isEmpty(args.doubles_number) then
+			table.insert(widgets, Title{name = 'Player Breakdown'})
+			table.insert(widgets, Cell{
+				name = 'Number of Players',
+				content = {args.player_number}
+			})
+			table.insert(widgets, Cell{
+				name = 'Doubles Players',
+				content = {args.doubles_number}
+			})
+		end
+	elseif id == 'prizepool' then
+		return {
+			Cell{
+				name = 'Prize pool',
+				content = {CustomLeague:_createPrizepool(args)}
+			},
+		}
+	elseif id == 'liquipediatier' then
+		return {
+			Cell{
+				name = 'Liquipedia Tier',
+				content = {CustomLeague:_createLiquipediaTierDisplay(args)}
+			},
+		}
+	end
+	return widgets
+end
+
+function CustomLeague:_createLiquipediaTierDisplay(args)
+	local tier = args.liquipediatier or ''
+	local tierType = args.liquipediatiertype or ''
+
+	if String.isEmpty(tier) then
+		return nil
+	end
+
+	local function buildTierString(tierString)
+		local tierText = Tier.text[tierString]
+		if not tierText then
+			table.insert(_league.warnings, tierString .. ' is not a known Liquipedia Tier/Tiertype')
+			return ''
+		else
+			return '[[' .. tierText .. ' Tournaments|' .. tierText .. ']]'
+		end
+	end
+	local tierDisplay = buildTierString(tier)
+	if String.isNotEmpty(tierType) then
+		tierDisplay = buildTierString(tierType) .. '&nbsp;(' .. tierDisplay .. ')'
+	end
+
+	return tierDisplay
+end
+
+function CustomLeague:_createPrizepool(args)
+	if String.isEmpty(args.prizepool) and String.isEmpty(args.prizepoolusd) then
+		return nil
+	end
+
+	local date
+	if String.isNotEmpty(args.currency_rate) then
+		date = args.currency_date
+	end
+
+	return PrizePoolCurrency._get({
+		prizepool = args.prizepool,
+		prizepoolusd = args.prizepoolusd,
+		currency = args.localcurrency,
+		rate = args.currency_rate,
+		date = date or Variables.varDefault('tournament_enddate', _TODAY),
+	})
+end
+
+function CustomLeague:defineCustomPageVariables(args)
+	-- Legacy vars
+	local sdate = Variables.varDefault('tournament_startdate', _TODAY)
+	local edate = Variables.varDefault('tournament_enddate', _TODAY)
+	Variables.varDefine('tournament_sdate', sdate)
+	Variables.varDefine('tournament_edate', edate)
+	Variables.varDefine('tournament_date', edate)
+
+	Variables.varDefine('tournament_ticker_name', args.tickername)
+	Variables.varDefine('tournament_tier', Variables.varDefault('tournament_liquipediatier'))
+	Variables.varDefine('tournament_valve_premier', args.valvepremier)
+	Variables.varDefine('tournament_link', mw.title.getCurrentTitle().prefixedText)
+end
+
+function CustomLeague:addToLpdb(lpdbData, args)
+	lpdbData['patch'] = args.patch
+	lpdbData['participantsnumber'] = args.team_number or args.player_number
+	lpdbData['extradata'] = {
+		region = args.region,
+		mode = args.mode,
+		participantsnumber =
+			not String.isEmpty(args.team_number) and args.team_number or args.player_number,
```

Comment:
participantsnumber is defined twice, once in extradata and once without. Is that correct?

Reply:
thx, i overlooked that^^